README.md

```diff
@@ -24,30 +26,40 @@ We use the Docker's building system and convert it at the end to a disk image th
 
 ##### 1. Making the environment
 
+> Note if you have an error when you try to connect on your Scaleway account, remove ~/.scwrc and retry
+
 ```console
 root@yourmachine> scw run --name="buildcowsay" builder
```

Comment:
use "image-builder" instead of "builder", it may conflict with a new image soon :)